src/backend/common/middleware.py

```diff
@@ -48,23 +49,21 @@ def __call__(self, environ: Any, start_response: Any):
         return self.app(environ, start_response)
 
 
-class AfterResponseMiddleware:
+class AfterResponseMiddleware(NdbMiddleware):
```

Comment:
Is there a better way of doing this so we don't have to repeat the ndb client / redis object creation?